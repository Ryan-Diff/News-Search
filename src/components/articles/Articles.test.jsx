import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Articles from './Articles';

describe('Articles component', () => {
    afterEach(() => cleanup());
    it('renders Articles', () => {
      const { asFragment } = render(<Articles
      articles={[{
          title:'Test Article',
          author:'Ryan Diff',
          description:'Super descriptive'
      }]}
      />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
