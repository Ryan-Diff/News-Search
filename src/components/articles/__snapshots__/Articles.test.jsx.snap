// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles component renders Articles 1`] = `
<DocumentFragment>
  <ul
    data-testid="articles"
  >
    <li>
      <h2>
        Test Article
      </h2>
      <p>
        Ryan Diff
      </p>
      <p>
        Super descriptive
      </p>
    </li>
  </ul>
</DocumentFragment>
`;
